lib/rucio/core/replica.py

```diff
@@ -799,6 +811,37 @@ def _list_replicas_for_files(file_clause, state_clause, files, rse_clause, updat
             {'scope': scope, 'name': name} in files and files.remove({'scope': scope, 'name': name})
 
 
+def get_multi_cache_prefix(cache_site, filename, logger=logging.log):
+
+    # print('Looking up prefix for cache:', cache_site, 'and filename:', filename)
+    x_caches = read_from_cache('CacheSites', expiration_time=60)
```

Comment:
please cache the error case where the http response is not successful, too. This would spam the log and the vp endpoint server address otherwise.

Reply:
agree. will fix